devices/forward-education/breakoutboardrelaypumpv10.json

```diff
@@ -0,0 +1,16 @@
+{
+  "id": "forward-education-breakoutboardrelaypumpv10",
+  "name": "Breakout Board + Relay + Pump",
+  "company": "Forward Education",
+  "description": "0x3412171b\n",
```

Comment:
do you have a better description?

Reply:
Whoops!  That was only meant to be for the product identifier that I couldnt plop into the form.

The description (for now) would be:
`The Climate Action Kit Breakout Board from Forward Education.  Use this board with your micro:bit and your sensor and motor modules to learn to code and build prototype hardware.`